Evaluation/Input/Content/Section2.2_Data.md

```diff
@@ -1,40 +1,38 @@
+| pK<sub>a</sub> (base)   |          | 5.18      | [Yoshikado 2018](#5-references)         | acid dissociation constant of conjugate base                 |
+| Solubility (pH7.4)      | mg/mL    | 0.03     | [DrugBank](#5-references)              | Aqueous Solubility in pH 7.4                 |
+| logP                    |          | 2.53     | [Yoshikado 2018](#5-references)         | Partition coefficient between octanol and water              |
+| fu                      | %        | 0.66       | [Yoshikado 2018](#5-references)   | Fraction unbound in plasma                                   |
+| K<sub>m</sub> OATP1B1   | µmol/L   | 0.13    | [Bednarczy 2016](#5-references)            | OATP1B1 Michaelis-Menten constant                             |
+| K<sub>m</sub> MRP2      | µmol/L   | 7.7      | [Gilibili 2017](#5-references)| MRP2 Michaelis-Menten                                    |
+| Synthesis rate        | nmol/h/kg  | 0.19     | [Mochizuki 2022](#5-references)          | Endogenous synthesis rate              |
+| Renal clearance           | L/h/kg | 0.04     | [Mochizuki 2022](#5-references)        | elimination rate via kidneys             |
 
-### Clinical Data  <a id="clinical-data"></a>
+### 2.2.2 Clinical data
 
-A literature search was performed to collect available clinical data on dapagliflozin in healthy adults.
+A literature search was performed to collect available clinical data on Coproporphyrin-I in adults. 
 
-#### Model Building <a id="model-building"></a>
+The following publications were found in adults for model building:
 
-The following studies were used for model building (training data):
+| Publication                   | Arm / Treatment / Information used for model building        |
+| :---------------------------- | :----------------------------------------------------------- |
+| [Mori 2020](#5-references)  | Baseline PK profiles in healthy subjects of Coproporphyrin-I|
+| [Takehara 2018](#5-references) | Baseline and DDI plasma PK profiles in healthy subjects of Coproporphyrin-I|
+| [Lai 2016](#5-references)   | Baseline and DDI plasma PK profiles in healthy subjects of Coproporphyrin-I|
+| [Zhang 2020](#5-references)   | Baseline and DDI plasma PK profiles in healthy subjects of Coproporphyrin-I|
 
-| Publication                 | Arm / Treatment / Information used for model building |
-| :-------------------------- | :---------------------------------------------------- |
-| [Dummy YEAR](#main-references) | Healthy Subjects with a single dose of X mg           |
-| ...                         | ...                                                   |
-| ...                         | ...                                                   |
-| ...                         | ...                                                   |
 
-#### Model Verification <a id="model-verification"></a>
+The following dosing scenarios were simulated and compared to respective data for model verification:
```

Comment:
This is now somehow without a context. I suggest to completely remove the following table with the DDI scenario, as these are not part of the evaluation and the simulations are not defined in the snapshot.